vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/service/StepDefinitionResolver.java

```diff
@@ -72,13 +72,17 @@ public StepDefinitionResolver(TextDocumentProvider textDocumentProvider)
     public Stream<ResolvedStepDefinition> resolveAtPosition(String documentIdentifier, Position position)
     {
         return findStep(textDocumentProvider.getTextDocument(documentIdentifier), position)
-                .map(step -> groupedStepDefinitions.get().get(step.getType())
-                        .stream()
-                        .map(tryToResolve(step))
-                        .flatMap(Optional::stream))
+                .map(step -> resolve(step, groupedStepDefinitions.get().get(step.getType()), false))
```

Comment:
```suggestion
                .flatMap(step -> resolve(step, groupedStepDefinitions.get().get(step.getType()), false))
```

Reply:
the initial strem here is from optional element so it won't work

Reply:
java.util.Optional#stream
?